### PR TITLE
ParseQueryAdapter for View Pager

### PR DESCRIPTION
**Pull request by pablobaxter**
I had attempted to do a pull request on this, but due to some issues, attempting to do it again on this account.  Here is the info:

This is a ParseQueryAdapter for use with ViewPager. Uses
FragmentStatePagerAdapter v13 under the hood, but works basically like
ParseQueryAdapter, except it attaches to a ViewPager.

This requires the minimum SDK to be raised to 13, due to the v13 support library usage. If needed, this can be modified to use the v4 support library quite easily.

Unit tests similar to that of ParseQueryAdapter, but modified for use with Fragments. Documentation also updated.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -43,3 +43,13 @@ gen-external-apklibs
 # Gradle
 .gradle
 build
+ParseUI-Widget/._src
```
Comment (wangmengyan95) [inline .gitignore]: This seems weird, why you have these files?

File: ParseUI-Widget/build.gradle [hunk 2]
```diff
@@ -39,7 +40,7 @@ android {
     buildToolsVersion "21.1.1"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 13
```
Comment (wangmengyan95) [inline ParseUI-Widget/build.gradle]: @stanleyw @grantland  Thoughts on this? Right now for Parse, the minSdkVersion is 9. What is our standard to deprecate a version?

File: ParseUI-Widget/build.gradle [hunk 3]
```diff
@@ -27,6 +27,7 @@ group = 'com.parse'
 
 dependencies {
     compile 'com.parse:parse-android:1.10.3'
+    compile 'com.android.support:support-v13:21.0.3'
```
Comment (wangmengyan95) [inline ParseUI-Widget/build.gradle]: Any reason you choose this version of support library?

**Comment by wangmengyan95** [comment]
Thanks for the PR @pablobaxter!! I am trying to fetch the PR and play with it. Since it is a large PR, it will take me some time to fully understand and review it. I will update my status here.

File: .gitignore (hunk 1 above)
Comment (pablobaxter) [inline .gitignore]: The drive I have my projects saved in, is an exFat file system, which automatically creates these files.  I just auto-ignore them, as they keep popping up.

File: ParseUI-Widget/build.gradle (hunk 3 above)
Comment (pablobaxter) [inline ParseUI-Widget/build.gradle]: Most of the projects I was working on were using native Fragments, so I went with the v13 support library.  However, I can easily make changes to this to use the v4 support library, or I can have 2 different versions of this adapter to ease development for those who have implemented v4 or native Fragments.

**Comment by grantland** [comment]
Thanks for your contribution, but we'd suggest using `ParseQueryPager` + `RecyclerView` for this use case (#129 #130).

**Pull request closed without merging**